ods_ci/libs/DataSciencePipelinesKfpTekton.py

```diff
@@ -96,24 +37,35 @@ def get_cert(self, api):
         cert_file.close()
         return file_name
 
+    def import_souce_code(self, path):
+        module_name = os.path.basename(path).replace('-', '_')
+        spec = importlib.util.spec_from_loader(
+            module_name,
+            importlib.machinery.SourceFileLoader(module_name, path)
+        )
+        module = importlib.util.module_from_spec(spec)
+        spec.loader.exec_module(module)
+        sys.modules[module_name] = module
+        return module
+
     @keyword
-    def kfp_tekton_create_run_from_pipeline_func(self, user, pwd, project, fn):
+    def kfp_tekton_create_run_from_pipeline_func(self, user, pwd, project, source_code, fn):
         client, _ = self.get_client(user, pwd, project)
-        # it is not a good idea use eval at all, but this is for testing purpose and make it easy the integration with
-        # the Robot Framework
-        # the fn parameter is without ()
-        # example: flipcoin_pipeline
-        pipeline = eval(fn)
+        # the current path is from where you are running the script
+        # sh ods_ci/run_robot_test.sh
+        # the current_path will be ods_ci
```

Comment:
I think the current path will be `ods-ci` rather than `ods_ci`

Reply:
Done